pkg/session/bootstrap.go

```diff
@@ -3365,6 +3371,14 @@ func upgradeToVer241(s sessiontypes.Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.default_roles ADD INDEX i_user (user)", dbterror.ErrDupKeyName)
 }
 
+func upgradeToVer242(s sessiontypes.Session, ver int64) {
+	if ver >= version242 {
+		return
+	}
+	doReentrantDDL(s, "ALTER TABLE mysql.tidb_global_task ADD COLUMN max_node_count INT AFTER `modify_params`;", infoschema.ErrColumnExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.tidb_global_task_history ADD COLUMN max_node_count INT AFTER `modify_params`;", infoschema.ErrColumnExists)
```

Comment:
Should we specify the default value for this column?

Reply:
The default value should be zero or a large integer so that it does not affect the behavior of an existing task.

Reply:
we can specify the default value be 0, so there is no need to check nil for tasks before upgrade